manifest.json

```diff
@@ -9,7 +9,9 @@
   },
 
   "background": {
-    "service_worker": "eventPage.js"
+    // Chrome only support service workers and Firefox only supports event pages (for now). Comment out the respective line for development
+    // "service_worker": "eventPage.js"
+    "scripts": ["eventPage.js"]
```

Comment:
This is an unfortunate limitation. It might be worthwhile to create two different directories in this repo or house the Firefox extension in a fork, because comments aren't valid JSON AFAIK.

Reply:
They are invalid JSON, but most parsers (including the Chrome and Firefox manifest one) don't seem to care. I had some issues linking to the source from a child directory, but I'll try that again. If we need to have a copy of the source at each spot, it probably is cleaner to just have a separate fork instead. I'll play around more and see what I can do

Reply:
I couldn't find a cleaner solution than having two copies of the source code (unless we want some build scripts). Since Firefox plans to eventually sort service workers, we will be able to combine them at one point